### ISSUE TITLE
Delete the Jitsi widget when everyone leaves

### ISSUE THREAD

**Issue by turt2live** [vector-im/riot-web#9055]

**Comment by Biep** [comment]
The widget ought to allow establishing a call - and not **be** a call.  It could also allow killing a call in progress.  One should not have to add an integration every time one wants to make a call.

**Comment by turt2live** [comment]
@Biep as mentioned in #riot, the "start call" button already does this

**Comment by Biep** [comment]
Yes, you are right - I completely misunderstood the nature of the widget.  As indicated in #riot-web, I still have some issues that may or may not be related.
